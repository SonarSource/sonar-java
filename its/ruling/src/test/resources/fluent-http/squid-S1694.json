{
'net.code-story:http:src/main/java/net/codestory/http/constants/Encodings.java':[
18,
],
'net.code-story:http:src/main/java/net/codestory/http/constants/FrameTypes.java':[
18,
],
'net.code-story:http:src/main/java/net/codestory/http/constants/Headers.java':[
18,
],
'net.code-story:http:src/main/java/net/codestory/http/constants/HttpStatus.java':[
18,
],
'net.code-story:http:src/main/java/net/codestory/http/constants/Methods.java':[
18,
],
'net.code-story:http:src/main/java/net/codestory/http/cors/CORSHelper.java':[
28,
],
}
