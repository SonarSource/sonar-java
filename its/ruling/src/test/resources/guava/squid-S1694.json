{
'com.google.guava:guava:src/com/google/common/collect/ForwardingImmutableList.java':[
27,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingImmutableMap.java':[
27,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingImmutableSet.java':[
27,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeCapture.java':[
29,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeVisitor.java':[
61,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AbstractFuture.java':[
834,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/GwtFuturesCatchingSpecialization.java':[
27,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Service.java':[
255,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/ServiceManager.java':[
146,
],
}
