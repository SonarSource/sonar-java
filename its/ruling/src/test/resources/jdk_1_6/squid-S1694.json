{
'jdk_1_6:java/awt/AttributeValue.java':[
12,
],
'jdk_1_6:java/awt/font/LayoutPath.java':[
25,
],
'jdk_1_6:java/awt/font/LineMetrics.java':[
29,
],
'jdk_1_6:java/awt/font/TextLine.java':[
465,
],
'jdk_1_6:java/io/ObjectInputStream.java':[
1056,
],
'jdk_1_6:java/io/ObjectOutputStream.java':[
858,
],
'jdk_1_6:java/lang/ref/Reference.java':[
24,
],
'jdk_1_6:java/net/Authenticator.java':[
43,
],
'jdk_1_6:java/net/CacheRequest.java':[
27,
],
'jdk_1_6:java/net/CacheResponse.java':[
25,
],
'jdk_1_6:java/net/SocketAddress.java':[
23,
],
'jdk_1_6:java/rmi/server/RMIClassLoaderSpi.java':[
45,
],
'jdk_1_6:java/security/AlgorithmParameterGeneratorSpi.java':[
38,
],
'jdk_1_6:java/security/AlgorithmParametersSpi.java':[
34,
],
'jdk_1_6:java/security/KeyFactorySpi.java':[
53,
],
'jdk_1_6:java/security/Policy.java':[
81,
],
'jdk_1_6:java/util/concurrent/ConcurrentHashMap.java':[
1057,
],
'jdk_1_6:java/util/concurrent/SynchronousQueue.java':[
137,
],
'jdk_1_6:java/util/jar/Pack200.java':[
95,
],
'jdk_1_6:java/util/prefs/FileSystemPreferences.java':[
349,
],
'jdk_1_6:java/util/regex/Pattern.java':[
5061,
],
}
