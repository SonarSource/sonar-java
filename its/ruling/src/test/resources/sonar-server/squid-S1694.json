{
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/computation/task/projectanalysis/issue/IssueVisitor.java':[
25,
],
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/permission/PermissionChange.java':[
31,
],
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/plugins/ServerExtensionInstaller.java':[
44,
],
}
