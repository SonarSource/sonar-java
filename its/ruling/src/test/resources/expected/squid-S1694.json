{
'project:com/sun/corba/se/PortableActivationIDL/ActivatorHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/BadServerDefinitionHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/EndPointInfoHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/EndpointInfoListHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/InitialNameServiceHelper.java':[
15,
],
'project:com/sun/corba/se/PortableActivationIDL/InitialNameServicePackage/NameAlreadyBoundHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/InvalidORBidHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/LocatorHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/LocatorPackage/ServerLocationPerORBHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/LocatorPackage/ServerLocationPerTypeHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/NoSuchEndPointHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBAlreadyRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBPortInfoHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBPortInfoListHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBProxyHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/ORBidListHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/RepositoryHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/RepositoryPackage/AppNamesHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/RepositoryPackage/ServerDefHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerAlreadyActiveHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerAlreadyInstalledHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerAlreadyRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerAlreadyUninstalledHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerHeldDownHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerIdsHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerManagerHelper.java':[
16,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerNotActiveHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerNotRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/PortableActivationIDL/ServerProxyHelper.java':[
14,
],
'project:com/sun/corba/se/PortableActivationIDL/TCPPortHelper.java':[
17,
],
'project:com/sun/corba/se/impl/corba/AnyImplHelper.java':[
36,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImplHelper.java':[
34,
],
'project:com/sun/corba/se/impl/encoding/CodeSetConversion.java':[
73,
109,
],
'project:com/sun/corba/se/impl/oa/poa/POAPolicyMediatorFactory.java':[
28,
],
'project:com/sun/corba/se/impl/orb/DataCollectorBase.java':[
470,
],
'project:com/sun/corba/se/impl/orb/DataCollectorFactory.java':[
34,
],
'project:com/sun/corba/se/impl/orbutil/RepositoryIdFactory.java':[
31,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubConnectImpl.java':[
52,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/AddressingDispositionHelper.java':[
36,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/IORAddressingInfoHelper.java':[
36,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/TargetAddressHelper.java':[
36,
],
'project:com/sun/corba/se/spi/activation/ActivatorHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/BadServerDefinitionHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/EndPointInfoHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/EndpointInfoListHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/InitialNameServiceHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/InitialNameServicePackage/NameAlreadyBoundHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/InvalidORBidHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/LocatorHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/LocatorPackage/ServerLocationHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/LocatorPackage/ServerLocationPerORBHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/NoSuchEndPointHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ORBAlreadyRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ORBPortInfoHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ORBPortInfoListHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ORBidHelper.java':[
13,
],
'project:com/sun/corba/se/spi/activation/ORBidListHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/POANameHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/RepositoryHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/RepositoryPackage/ServerDefHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/RepositoryPackage/StringSeqHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyActiveHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyInstalledHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerAlreadyUninstalledHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerHeldDownHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerHelper.java':[
14,
],
'project:com/sun/corba/se/spi/activation/ServerIdHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerIdsHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerManagerHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerNotActiveHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/ServerNotRegisteredHelper.java':[
11,
],
'project:com/sun/corba/se/spi/activation/TCPPortHelper.java':[
11,
],
'project:com/sun/corba/se/spi/copyobject/CopyobjectDefaults.java':[
35,
],
'project:com/sun/corba/se/spi/ior/iiop/IIOPFactories.java':[
69,
],
'project:com/sun/corba/se/spi/logging/CORBALogDomains.java':[
57,
],
'project:com/sun/corba/se/spi/logging/LogWrapperBase.java':[
32,
],
'project:com/sun/corba/se/spi/orb/OperationFactory.java':[
70,
],
'project:com/sun/corba/se/spi/orbutil/closure/ClosureFactory.java':[
31,
],
'project:com/sun/corba/se/spi/orbutil/proxy/DelegateInvocationHandlerImpl.java':[
39,
],
'project:com/sun/corba/se/spi/presentation/rmi/PresentationDefaults.java':[
36,
],
'project:com/sun/corba/se/spi/presentation/rmi/StubAdapter.java':[
59,
],
'project:com/sun/corba/se/spi/transport/TransportDefault.java':[
45,
],
'project:com/sun/javadoc/Doclet.java':[
51,
],
'project:com/sun/org/apache/bcel/internal/Repository.java':[
75,
],
'project:com/sun/org/apache/bcel/internal/classfile/Utility.java':[
72,
],
'project:com/sun/org/apache/bcel/internal/util/ClassPath.java':[
307,
],
'project:com/sun/org/apache/xalan/internal/lib/ExsltBase.java':[
34,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java':[
1751,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDAbstractTraverser.java':[
63,
],
'project:com/sun/org/apache/xerces/internal/parsers/XMLParser.java':[
52,
],
'project:com/sun/org/apache/xerces/internal/util/AugmentationsImpl.java':[
104,
],
'project:com/sun/org/apache/xml/internal/resolver/helpers/FileURL.java':[
46,
],
'project:com/sun/org/apache/xml/internal/resolver/helpers/PublicId.java':[
36,
],
'project:com/sun/org/apache/xml/internal/security/encryption/XMLCipher.java':[
3336,
],
'project:com/sun/org/apache/xml/internal/security/keys/keyresolver/KeyResolverSpi.java':[
45,
],
'project:com/sun/org/apache/xml/internal/security/keys/storage/StorageResolverSpi.java':[
32,
],
'project:com/sun/org/apache/xml/internal/utils/Hashtree2Node.java':[
44,
],
'project:com/sun/org/apache/xml/internal/utils/XMLStringFactory.java':[
28,
],
'project:com/sun/org/apache/xpath/internal/objects/XNodeSet.java':[
733,
],
'project:com/sun/source/util/JavacTask.java':[
45,
],
'project:java/awt/AttributeValue.java':[
30,
],
'project:java/awt/font/LayoutPath.java':[
43,
],
'project:java/awt/font/LineMetrics.java':[
44,
],
'project:java/awt/font/TextLine.java':[
478,
],
'project:java/io/ObjectInputStream.java':[
1075,
],
'project:java/io/ObjectOutputStream.java':[
874,
],
'project:java/lang/invoke/MethodHandle.java':[
392,
],
'project:java/lang/invoke/MethodHandleImpl.java':[
47,
],
'project:java/lang/ref/Reference.java':[
41,
],
'project:java/net/Authenticator.java':[
60,
],
'project:java/net/CacheRequest.java':[
44,
],
'project:java/net/CacheResponse.java':[
42,
],
'project:java/net/SocketAddress.java':[
41,
],
'project:java/rmi/server/RMIClassLoaderSpi.java':[
62,
],
'project:java/security/AlgorithmParameterGeneratorSpi.java':[
55,
],
'project:java/security/AlgorithmParametersSpi.java':[
51,
],
'project:java/security/KeyFactorySpi.java':[
70,
],
'project:java/security/Policy.java':[
97,
],
'project:java/util/concurrent/ConcurrentHashMap.java':[
1251,
],
'project:java/util/concurrent/SynchronousQueue.java':[
166,
],
'project:java/util/jar/Pack200.java':[
109,
],
'project:java/util/prefs/FileSystemPreferences.java':[
362,
],
'project:java/util/regex/Pattern.java':[
5457,
],
'project:javax/accessibility/AccessibleBundle.java':[
50,
],
'project:javax/accessibility/AccessibleHyperlink.java':[
46,
],
'project:javax/imageio/IIOParam.java':[
49,
],
'project:javax/sound/midi/spi/MidiFileReader.java':[
45,
],
'project:javax/sound/midi/spi/SoundbankReader.java':[
47,
],
'project:javax/sound/sampled/Control.java':[
44,
],
'project:javax/sound/sampled/spi/AudioFileReader.java':[
45,
],
'project:javax/swing/JFormattedTextField.java':[
863,
],
'project:javax/swing/filechooser/FileFilter.java':[
51,
],
'project:javax/swing/filechooser/FileView.java':[
68,
],
'project:javax/swing/plaf/ComponentUI.java':[
54,
],
'project:javax/swing/plaf/synth/SynthPainter.java':[
62,
],
'project:javax/swing/table/TableStringConverter.java':[
34,
],
'project:javax/swing/text/DocumentFilter.java':[
132,
],
'project:javax/swing/text/LayeredHighlighter.java':[
58,
],
'project:javax/swing/text/NavigationFilter.java':[
123,
],
'project:javax/swing/text/html/HTMLDocument.java':[
1845,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
969,
],
'project:javax/swing/tree/AbstractLayoutCache.java':[
509,
],
'project:javax/xml/bind/Binder.java':[
81,
],
'project:javax/xml/bind/Marshaller.java':[
773,
],
'project:javax/xml/bind/SchemaOutputResolver.java':[
46,
],
'project:javax/xml/bind/Unmarshaller.java':[
1086,
],
'project:javax/xml/bind/WhiteSpaceProcessor.java':[
33,
],
'project:javax/xml/ws/EndpointContext.java':[
47,
],
'project:javax/xml/ws/spi/Invoker.java':[
49,
],
'project:javax/xml/ws/spi/http/HttpHandler.java':[
41,
],
'project:org/omg/CORBA/AnySeqHelper.java':[
50,
],
'project:org/omg/CORBA/BooleanSeqHelper.java':[
50,
],
'project:org/omg/CORBA/CharSeqHelper.java':[
50,
],
'project:org/omg/CORBA/CompletionStatusHelper.java':[
37,
],
'project:org/omg/CORBA/Context.java':[
102,
],
'project:org/omg/CORBA/ContextList.java':[
58,
],
'project:org/omg/CORBA/CurrentHelper.java':[
38,
],
'project:org/omg/CORBA/DefinitionKindHelper.java':[
38,
],
'project:org/omg/CORBA/DoubleSeqHelper.java':[
50,
],
'project:org/omg/CORBA/Environment.java':[
37,
],
'project:org/omg/CORBA/ExceptionList.java':[
54,
],
'project:org/omg/CORBA/FieldNameHelper.java':[
37,
],
'project:org/omg/CORBA/FloatSeqHelper.java':[
50,
],
'project:org/omg/CORBA/IDLTypeHelper.java':[
38,
],
'project:org/omg/CORBA/IdentifierHelper.java':[
38,
],
'project:org/omg/CORBA/LongLongSeqHelper.java':[
50,
],
'project:org/omg/CORBA/LongSeqHelper.java':[
50,
],
'project:org/omg/CORBA/NVList.java':[
93,
],
'project:org/omg/CORBA/NameValuePairHelper.java':[
37,
],
'project:org/omg/CORBA/NamedValue.java':[
96,
],
'project:org/omg/CORBA/ObjectHelper.java':[
41,
],
'project:org/omg/CORBA/OctetSeqHelper.java':[
50,
],
'project:org/omg/CORBA/ParameterModeHelper.java':[
20,
],
'project:org/omg/CORBA/PolicyErrorCodeHelper.java':[
17,
],
'project:org/omg/CORBA/PolicyErrorHelper.java':[
16,
],
'project:org/omg/CORBA/PolicyHelper.java':[
38,
],
'project:org/omg/CORBA/PolicyListHelper.java':[
38,
],
'project:org/omg/CORBA/PolicyTypeHelper.java':[
40,
],
'project:org/omg/CORBA/RepositoryIdHelper.java':[
39,
],
'project:org/omg/CORBA/Request.java':[
67,
],
'project:org/omg/CORBA/ServiceDetailHelper.java':[
35,
],
'project:org/omg/CORBA/ServiceInformationHelper.java':[
35,
],
'project:org/omg/CORBA/SetOverrideTypeHelper.java':[
37,
],
'project:org/omg/CORBA/ShortSeqHelper.java':[
50,
],
'project:org/omg/CORBA/StringSeqHelper.java':[
13,
],
'project:org/omg/CORBA/StructMemberHelper.java':[
38,
],
'project:org/omg/CORBA/TypeCode.java':[
170,
],
'project:org/omg/CORBA/ULongLongSeqHelper.java':[
50,
],
'project:org/omg/CORBA/ULongSeqHelper.java':[
50,
],
'project:org/omg/CORBA/UShortSeqHelper.java':[
50,
],
'project:org/omg/CORBA/UnionMemberHelper.java':[
38,
],
'project:org/omg/CORBA/UnknownUserExceptionHelper.java':[
38,
],
'project:org/omg/CORBA/ValueBaseHelper.java':[
43,
],
'project:org/omg/CORBA/ValueMemberHelper.java':[
38,
],
'project:org/omg/CORBA/VersionSpecHelper.java':[
38,
],
'project:org/omg/CORBA/VisibilityHelper.java':[
40,
],
'project:org/omg/CORBA/WCharSeqHelper.java':[
50,
],
'project:org/omg/CORBA/WStringSeqHelper.java':[
13,
],
'project:org/omg/CORBA/WrongTransactionHelper.java':[
37,
],
'project:org/omg/CosNaming/BindingHelper.java':[
11,
],
'project:org/omg/CosNaming/BindingIteratorHelper.java':[
20,
],
'project:org/omg/CosNaming/BindingListHelper.java':[
15,
],
'project:org/omg/CosNaming/BindingTypeHelper.java':[
16,
],
'project:org/omg/CosNaming/IstringHelper.java':[
11,
],
'project:org/omg/CosNaming/NameComponentHelper.java':[
11,
],
'project:org/omg/CosNaming/NameHelper.java':[
15,
],
'project:org/omg/CosNaming/NamingContextExtHelper.java':[
25,
],
'project:org/omg/CosNaming/NamingContextExtPackage/AddressHelper.java':[
15,
],
'project:org/omg/CosNaming/NamingContextExtPackage/InvalidAddressHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextExtPackage/StringNameHelper.java':[
16,
],
'project:org/omg/CosNaming/NamingContextExtPackage/URLStringHelper.java':[
16,
],
'project:org/omg/CosNaming/NamingContextHelper.java':[
21,
],
'project:org/omg/CosNaming/NamingContextPackage/AlreadyBoundHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextPackage/CannotProceedHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextPackage/InvalidNameHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextPackage/NotEmptyHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextPackage/NotFoundHelper.java':[
11,
],
'project:org/omg/CosNaming/NamingContextPackage/NotFoundReasonHelper.java':[
15,
],
'project:org/omg/DynamicAny/AnySeqHelper.java':[
11,
],
'project:org/omg/DynamicAny/DynAnyFactoryHelper.java':[
33,
],
'project:org/omg/DynamicAny/DynAnyFactoryPackage/InconsistentTypeCodeHelper.java':[
11,
],
'project:org/omg/DynamicAny/DynAnyHelper.java':[
79,
],
'project:org/omg/DynamicAny/DynAnyPackage/InvalidValueHelper.java':[
11,
],
'project:org/omg/DynamicAny/DynAnyPackage/TypeMismatchHelper.java':[
11,
],
'project:org/omg/DynamicAny/DynAnySeqHelper.java':[
11,
],
'project:org/omg/DynamicAny/DynArrayHelper.java':[
17,
],
'project:org/omg/DynamicAny/DynEnumHelper.java':[
16,
],
'project:org/omg/DynamicAny/DynFixedHelper.java':[
17,
],
'project:org/omg/DynamicAny/DynSequenceHelper.java':[
15,
],
'project:org/omg/DynamicAny/DynStructHelper.java':[
16,
],
'project:org/omg/DynamicAny/DynUnionHelper.java':[
22,
],
'project:org/omg/DynamicAny/DynValueHelper.java':[
24,
],
'project:org/omg/DynamicAny/FieldNameHelper.java':[
11,
],
'project:org/omg/DynamicAny/NameDynAnyPairHelper.java':[
11,
],
'project:org/omg/DynamicAny/NameDynAnyPairSeqHelper.java':[
11,
],
'project:org/omg/DynamicAny/NameValuePairHelper.java':[
11,
],
'project:org/omg/DynamicAny/NameValuePairSeqHelper.java':[
11,
],
'project:org/omg/IOP/CodecFactoryHelper.java':[
17,
],
'project:org/omg/IOP/CodecFactoryPackage/UnknownEncodingHelper.java':[
11,
],
'project:org/omg/IOP/CodecPackage/FormatMismatchHelper.java':[
11,
],
'project:org/omg/IOP/CodecPackage/InvalidTypeForEncodingHelper.java':[
11,
],
'project:org/omg/IOP/CodecPackage/TypeMismatchHelper.java':[
11,
],
'project:org/omg/IOP/ComponentIdHelper.java':[
16,
],
'project:org/omg/IOP/IORHelper.java':[
11,
],
'project:org/omg/IOP/MultipleComponentProfileHelper.java':[
13,
],
'project:org/omg/IOP/ProfileIdHelper.java':[
13,
],
'project:org/omg/IOP/ServiceContextHelper.java':[
11,
],
'project:org/omg/IOP/ServiceContextListHelper.java':[
13,
],
'project:org/omg/IOP/ServiceIdHelper.java':[
13,
],
'project:org/omg/IOP/TaggedComponentHelper.java':[
11,
],
'project:org/omg/IOP/TaggedProfileHelper.java':[
11,
],
'project:org/omg/Messaging/SyncScopeHelper.java':[
13,
],
'project:org/omg/PortableInterceptor/AdapterManagerIdHelper.java':[
18,
],
'project:org/omg/PortableInterceptor/AdapterNameHelper.java':[
15,
],
'project:org/omg/PortableInterceptor/AdapterStateHelper.java':[
15,
],
'project:org/omg/PortableInterceptor/CurrentHelper.java':[
29,
],
'project:org/omg/PortableInterceptor/ForwardRequestHelper.java':[
11,
],
'project:org/omg/PortableInterceptor/IORInterceptor_3_0Helper.java':[
11,
],
'project:org/omg/PortableInterceptor/InvalidSlotHelper.java':[
11,
],
'project:org/omg/PortableInterceptor/ORBIdHelper.java':[
13,
],
'project:org/omg/PortableInterceptor/ORBInitInfoPackage/DuplicateNameHelper.java':[
11,
],
'project:org/omg/PortableInterceptor/ORBInitInfoPackage/InvalidNameHelper.java':[
11,
],
'project:org/omg/PortableInterceptor/ORBInitInfoPackage/ObjectIdHelper.java':[
13,
],
'project:org/omg/PortableInterceptor/ObjectIdHelper.java':[
15,
],
'project:org/omg/PortableInterceptor/ObjectReferenceFactoryHelper.java':[
15,
],
'project:org/omg/PortableInterceptor/ObjectReferenceTemplateHelper.java':[
19,
],
'project:org/omg/PortableInterceptor/ObjectReferenceTemplateSeqHelper.java':[
15,
],
'project:org/omg/PortableInterceptor/ServerIdHelper.java':[
13,
],
'project:org/omg/PortableServer/CurrentHelper.java':[
55,
],
'project:org/omg/PortableServer/CurrentPackage/NoContextHelper.java':[
11,
],
'project:org/omg/PortableServer/ForwardRequestHelper.java':[
11,
],
'project:org/omg/PortableServer/POAHelper.java':[
46,
],
'project:org/omg/PortableServer/POAManagerPackage/AdapterInactiveHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/AdapterAlreadyExistsHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/AdapterNonExistentHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/InvalidPolicyHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/NoServantHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/ObjectAlreadyActiveHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/ObjectNotActiveHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/ServantAlreadyActiveHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/ServantNotActiveHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/WrongAdapterHelper.java':[
11,
],
'project:org/omg/PortableServer/POAPackage/WrongPolicyHelper.java':[
11,
],
'project:org/omg/PortableServer/ServantActivatorHelper.java':[
16,
],
'project:org/omg/PortableServer/ServantLocatorHelper.java':[
30,
],
}
